internal/pb/s2.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.36.2
+// 	protoc-gen-go v1.36.3
```

Comment:
is this auto generated, or did you bump something

Reply:
Auto-generated (Tooling upgrade, we're always using latest protoc)